### PR TITLE
Example to install Office 365 with Commands Gallery 

### PR DESCRIPTION
**Pull request by jworkmanjc**
@joffems I'm opening your pull request on this branch and made a few changes to your script, once I get another set of eyes on this we'll approve. Thank you for contributing.

### CODE REVIEW

File: PowerShell/JumpCloud Commands Gallery/Mac Commands/Application Installs/Mac - Install Microsoft Office.md [hunk 1]
```diff
@@ -122,3 +110,20 @@ echo "Installed Microsoft Office"
...
+It's advised to set a longer timeout window for this command, at the time of writing the Office 365 Installer was 1.5GB. Download and install time must be taken into consideration. In testing, a timeout of 800 seconds proved adequate.
+
+This is a community written command from [joffems](https://github.com/joffems) thanks for contributing!
+
+#### *Import This Command*
+
+To import this command into your JumpCloud tenant run the below command using the [JumpCloud PowerShell Module](https://github.com/TheJumpCloud/support/wiki/Installing-the-JumpCloud-PowerShell-Module)
+
+```
+Import-JCCommand -URL 'https://REPLACEME'
```
Comment (ahendrickson-jc) [inline PowerShell/JumpCloud Commands Gallery/Mac Commands/Application Installs/Mac - Install Microsoft Office.md:128]: Do we need to replace "REPLACEME" with:
Mac - Install Microsoft Office 365 | v1.0 JCCG

File: PowerShell/JumpCloud Commands Gallery/Mac Commands/Application Installs/Mac - Install Microsoft Office.md [hunk 2]
```diff
@@ -122,3 +110,20 @@ echo "Installed Microsoft Office"
 rm -r /tmp/$TempFolder
```
Comment (ahendrickson-jc) [inline PowerShell/JumpCloud Commands Gallery/Mac Commands/Application Installs/Mac - Install Microsoft Office.md:110]: Would we want `rm -rf` here, just to suppress any messages?

File: PowerShell/JumpCloud Commands Gallery/Mac Commands/Application Installs/Mac - Install Microsoft Office.md [hunk 3]
```diff
@@ -73,34 +76,19 @@ fi
...
-else
-
-    echo "Microsoft Outlook not present in /Applications folder"
-fi
+list=("Microsoft Excel" "Microsoft OneNote" "Microsoft Outlook" "Microsoft PowerPoint" "Microsoft Word")
+for n in "${list[@]}"; do
+    app="/Applications/$n.app"
+    if [[ -d "$app" ]]; then
+        echo "$n already present in /Applications folder"
+        rm -r /tmp/$TempFolder
```
Comment (ahendrickson-jc) [inline PowerShell/JumpCloud Commands Gallery/Mac Commands/Application Installs/Mac - Install Microsoft Office.md:84]: Would we want `rm -rf` here, just to suppress any messages?

File: PowerShell/JumpCloud Commands Gallery/Mac Commands/Application Installs/Mac - Install Microsoft Office.md (hunk 1 above)
Comment (jworkmanjc) [reply PowerShell/JumpCloud Commands Gallery/Mac Commands/Application Installs/Mac - Install Microsoft Office.md:128]: Long story short, we replace the link *after* is published to master with a shortened URL

File: PowerShell/JumpCloud Commands Gallery/Mac Commands/Application Installs/Mac - Install Microsoft Office.md (hunk 3 above)
Comment (jworkmanjc) [reply PowerShell/JumpCloud Commands Gallery/Mac Commands/Application Installs/Mac - Install Microsoft Office.md:84]: I think that's fine, it works without but that always assumes that the tmp/$TempFolder exists.

Comment (jworkmanjc) [reply PowerShell/JumpCloud Commands Gallery/Mac Commands/Application Installs/Mac - Install Microsoft Office.md:84]: The only difference would be that rm -r would return:
rm: /tmp/theFOlderName : No such file or directory
which may be useful for diagnosis. Either really works in this case I believe.

**Pull request merged**